go-controller/pkg/config/config.go

```diff
@@ -226,6 +226,8 @@ type GatewayConfig struct {
 	VLANID uint `gcfg:"vlan-id"`
 	// NodeportEnable sets whether to provide Kubernetes NodePort service or not
 	NodeportEnable bool `gcfg:"nodeport"`
+	// DisableSNATExternalGWs sets whether to disable SNAT traffic or not for pods in namespace annotated with routing-external-gws
```

Comment:
"whether to disable SNAT traffic" makes it sound like this is blocking traffic that has been SNATted.

"DisableSNATExternalGWs sets whether to disable SNATting of external traffic in namespaces annotated with routing-external-gws"

Reply:
ack, will fix